core/templates/dev/head/domain/exploration/AnswerGroupObjectFactory.js

```diff
@@ -20,31 +20,34 @@
 oppia.factory('AnswerGroupObjectFactory', [
   'RuleObjectFactory', 'OutcomeObjectFactory',
   function(RuleObjectFactory, OutcomeObjectFactory) {
-    var AnswerGroup = function(rules, outcome) {
+    var AnswerGroup = function(rules, outcome, trainingData) {
       this.rules = rules;
       this.outcome = outcome;
+      this.training_data = trainingData;
```

Comment:
this.trainingData. Only use underscores for backend stuff (and toBackendDict()).

Reply:
Done